src/safeposix/dispatcher.rs

```diff
@@ -382,7 +382,7 @@ pub extern "C" fn dispatcher(cageid: u64, callnum: i32, arg1: Arg, arg2: Arg, ar
             check_and_dispatch!(cage.gethostname_syscall, interface::get_mutcbuf(arg1), interface::get_isize(arg2))
         }
         MKDIR_SYSCALL => {
-            check_and_dispatch!(cage.mkdir_syscall, interface::get_cstr(arg1), interface::get_uint(arg2))
+            check_and_dispatch!(cage.mkdir_syscall, interface::get_cstr(arg1), interface::get_uint(arg2), Ok::<Option<&FilesystemMetadata>, i32>(None))
```

Comment:
I'm assuming this is for lindfs to work properly, which is fine, but leaving a comment would probably be helpful for us in the future when were scratching our heads.